### PR TITLE
[BugFix] Handling malformed/wrong config name in explore.py

### PR DESCRIPTION
**Pull request by pkundurthy**
## Summary

- Users reported seeing the following errors when trying to setup locally for first time. 

```
teams=teams)
  File "/opt/homebrew/bin/explore.py", line 145, in build_entry
    (team, conf_module) = entry["name"][0].split(".", 1)
ValueError: not enough values to unpack (expected 2, got 1)
```

The problematic input was a string of the form
`my_long_group_by_name_without_periods`
instead of the expected
`team_namespace.my_group_by_name`

We opt to return **None** for malformed inputs like this. 

- Also adding an Ignore condition for flake for E203
See this open bug:
https://github.com/PyCQA/pycodestyle/issues/373#issuecomment-760190686

## Why / Goal

- Unblock new users running through the first-time setup and `explore.py` commands. 

## Test Plan

- [ ] Added Unit Tests
- [X] Covered by existing CI
- [ ] Integration tested

## Checklist
- [ ] Documentation update

### CODE REVIEW

File: api/py/ai/chronon/repo/explore.py [hunk 1]
```diff
@@ -138,7 +119,8 @@ def build_entry(conf, index_spec, conf_type, root=CWD, teams=None):
             result.extend(extract_json(path, conf_dict))
         entry[column] = result
 
-    if len(entry["name"]) == 0:
+    # if no name or name does not conform to the expected <team>.<config>. format return None
+    if len(entry["name"]) == 0 or len(entry["name"][0].split(".")) < 2:
```
Comment (pkundurthy) [inline api/py/ai/chronon/repo/explore.py:123]: this is the only change; the rest are linting updates

File: .circleci/config.yml [hunk 2]
```diff
@@ -102,7 +102,7 @@ jobs:
                       conda activate chronon_py
                       cd /chronon/api/py/ai/chronon
                       pip install importlib-metadata==4.11.4 #Install importlib-metadata < 5
-                      flake8 --extend-ignore=W605,Q000,F631
+                      flake8 --extend-ignore=W605,Q000,F631,E203
```
Comment (pkundurthy) [inline .circleci/config.yml:105]: see note on open flake bug in the description

**Review by pengyu-hou — APPROVED** [review]
Awesome!

**Review by hzding621 — APPROVED** [review]
thanks for running the lint and improving the code quality!

**Pull request merged**